snap/snapcraft.yaml

```diff
@@ -21,48 +21,62 @@ compression: lzo
 apps:
   quickbib:
     command: usr/bin/quickbib
+    desktop: usr/share/applications/io.github.archisman_panigrahi.QuickBib.desktop
     environment:
-      PYTHONPATH: $SNAP/usr/lib/python3/site-packages:$SNAP/usr/lib/python3/dist-packages:$PYTHONPATH
-      QT_PLUGIN_PATH: ${SNAP}/usr/lib/x86_64-linux-gnu/qt6/plugins
-      QT_QPA_PLATFORM_PLUGIN_PATH: ${SNAP}/usr/lib/x86_64-linux-gnu/qt6/plugins/platforms
-      # SNAP_DESKTOP_RUNTIME: $SNAP/kf6
-      # QT_VERSION: "6"
-      # XDG_DATA_DIRS: $SNAP/usr/share:${XDG_DATA_DIRS:+:$XDG_DATA_DIRS}
-    plugs:
-      - network
-      - x11
-      - wayland
+      PYTHONPATH: $SNAP/usr/lib/python3/dist-packages:$PYTHONPATH
+      QT_QPA_PLATFORMTHEME: xdgdesktopportal
+      LD_LIBRARY_PATH: $SNAP/lxqt-support/usr/lib/$CRAFT_ARCH_TRIPLET_BUILD_FOR:$LD_LIBRARY_PATH
+      QT_PLUGIN_PATH: $SNAP/lxqt-support/usr/lib/$CRAFT_ARCH_TRIPLET_BUILD_FOR/qt6/plugins:$QT_PLUGIN_PATH
+    extensions: [kde-neon-6]
+
+plugs:
+  lxqt-support:
+    content: lxqt-support
+    interface: content
+    target: $SNAP/lxqt-support
+    default-provider: lxqt-support
+
+layout:
+  /usr/share/color-schemes:
+    symlink: $SNAP/lxqt-support/usr/share/color-schemes
+  /usr/share/Kvantum:
+    symlink: $SNAP/lxqt-support/usr/share/Kvantum
+
+package-repositories:
+  - type: apt
+    url: https://ppa.launchpadcontent.net/apandada1/quickbib/ubuntu
+    suites: [noble]
+    components: [main]
+    architectures: [arm64, amd64]
```

Comment:
Can we also add other supported architectures? Just let me know if that would directly work, if yes, I will take care of it after merging.

<img width="812" height="278" alt="Image" src="https://github.com/user-attachments/assets/5a5014de-6f9a-4422-a486-7bba7e2f9f37" />